hack/run-test-serial-e2e.sh

```diff
@@ -156,7 +156,8 @@ function runtests() {
 		--ginkgo.junit-report=${REPORT_FILE} \
 		${NO_COLOR} \
 		${SKIP} \
-		${FOCUS}
+		${FOCUS} \
+		--ginkgo.timeout="5h"
```

Comment:
the fix makes sense. Please move it just after `--ginkgo.v` only because cosmetic reasons (no functional change)

Reply:
done, thanks!